dist/js/app.js

```diff
@@ -136,7 +210,9 @@ function setDownloadSection() {
       }
       else { // if there IS a matching binary for the user's OS:
         var fullOSName = OS; // defaults this variable to be the detected OS name
-        if(OS == "Win") {
+        if(OS == "Linux") {
+          fullOSName = "Linux x86-64"; // add 'x86-64'
+        } else if(OS == "Win") {
```

Comment:
Is it always Linux x86-64?  In general I wonder if this functionality can be replaced with a well known borwser and O/S detection lib.